models/skillImpactsModel.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../database/dbConfig');
+
+
+async function findSkills(campaign_id) {
+    const skills = await db('skilled_impact_requests')
+        .where({campaign_id})
+        .join('project_goals', 'project_goals.skilled_impact_request_id', 'skilled_impact_requests.id')
+        .select(
+            '*'
+        )
+        .first();
+    return skills;
```

Comment:
We talked about this on Friday but how's the data going to look if we immediately return? `skills` probably needs to be reformatted into something like
```javascript
{
  skilled_impact_requests: [
    drone: {
      point_of_contact: ...,
      welcome_message: ...,
      our_contribution: ...,
      project_goals: [
        {
          goal_title: ...,
          description: ...,
        }
      ]
    },
    photography: { ... }
  ]
}
```
where "drone" and "photography" are placeholders for the actual skills requested for that `campaign_id`.